src/sync.cpp

```diff
@@ -1301,7 +1301,7 @@ LocalNode* Sync::checkpath(LocalNode* l, string* localpath, string* localname, d
         isroot = l == localroot.get() && !newname.size();
     }
 
-    LOG_verbose << "Scanning: " << path;
+    LOG_verbose << "Scanning: " << path << " in=" << initializing << " full=" << fullscan << " l=" << l;
```

Comment:
`if (localname == nullptr) then l == nullptr`. Is it safe to pass `l` to the logger in that case?

Reply:
yes, it'll print `NULL` or something similar